xchange-okcoin/src/main/java/com/xeiam/xchange/okcoin/service/polling/trade/OkCoinTradeServiceRaw.java

```diff
@@ -1,11 +1,11 @@
-package com.xeiam.xchange.okcoin.service.polling;
+package com.xeiam.xchange.okcoin.service.polling.trade;
```

Comment:
I would keep it in the original package, i.e. pooling. Otherwise it brings incompatibility with the older version. I would especially avoid doing this change in 'pooling' since our work is related to 'streaming'